### PR TITLE
[StorageClasses] Add gp3 to available ebs volume types

### PR DESCRIPTION
**Pull request by benjaminguttmann-avtq**
Adding information about gp3 to ebs storage class documentation.

### CODE REVIEW

**Comment by sftim** [comment]
/sig storage

This change would need technical review.

**Comment by geoffcline** [comment]
reviewed, this is consistent with AWS behavior. 

/lgtm

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -271,9 +271,9 @@ parameters:
   fsType: ext4
 ```
 
-* `type`: `io1`, `gp2`, `sc1`, `st1`. See
+* `type`: `io1`, `gp2`, `gp3`, `sc1`, `st1`. See
   [AWS docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html)
-  for details. Default: `gp2`.
+  for details. Default: `gp3`.
```
Comment (sftim) [inline content/en/docs/concepts/storage/storage-classes.md:276]: How do we know this is the default _in the Kubernetes API_?

Comment (benjaminguttmann-avtq) [reply content/en/docs/concepts/storage/storage-classes.md:276]: Took the information from here => https://github.com/kubernetes-sigs/aws-ebs-csi-driver#createvolume-parameters

**Comment by sftim** [comment]
This has prompted https://github.com/kubernetes/website/issues/33036

Was that change wrong?

File: content/en/docs/concepts/storage/storage-classes.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/storage/storage-classes.md:276]: I noticed that https://github.com/kubernetes-sigs/aws-ebs-csi-driver#createvolume-parameters is actually documenting a different (out of tree) storage driver.

**Comment by sftim** [comment]
Also
/sig cloud-provider

**Comment by geoffcline** [comment]
I think it's technically wrong, but the page is confusing about which driver it refers. Perhaps we should more clearly identify when we are talking in-tree/out of tree.

**Pull request merged**